artpaint/controls/ColorSlider.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2022, Dale Cieslak
+ * Distributed under the terms of the MIT License.
+ *
+ * Authors:
+ * 		Dale Cieslak <dcieslak@yahoo.com>
+ *
+ */
+
+#ifndef COLOR_SLIDER_H
+#define COLOR_SLIDER_H
+
+
+#include <Bitmap.h>
+#include <Slider.h>
+
+
+class ColorSlider: public BSlider
+{
+public:
+ 								ColorSlider(BRect frame, const char* name,
```

Comment:
Why are these constructors not aligned with the other methods?

Reply:
That’s a good question. Not sure what I did there…?